sphinx/frontend-bundling.rst

```diff
@@ -0,0 +1,20 @@
+Frontend Bundling
+=================
+
+Webpack is being used to generate frontend bundles based on TypeScript code that is compiled to JavaScript as well as SCSS code which is being transpiled to CSS. The following bundles are defined:
+
+1. `editor.js/css`: Contains TinyMCE and related assets. It was decided to bundle TinyMCE in a separate file because it is the largest of all bundled assets
+2. `main.js/css`: Contains everything else
+
+To get started with adding new code go to `src/frontend` and open `index.ts` or `editor.ts` (depending on the bundle that should be changed).
+In this file you can import any TypeScript or JavaScript file or library that is needed for the application.
+Generated assets will be stored in `src/cms/static/generated`. When running `dev-tools/run.sh`, `webpack-dev-server` is being used to generate JavaScript that includes CSS as well as SourceMaps to help debugging the output. For a production build one should run `npm run prod`. This will generate minified js and css files in the same target directory.
+
+Compatible Browsers
+===================
+
+We use `babel-preset-env` as well als `postcss-preset-env` to transpile or polyfill css and js features for supported browsers. `.browserslistrc` contains a list of supported browsers (mostly selected by looking at their usage stats and support status).
+
+.. Note::
+
+    At some point IE 11 will probably be removed from that list.
```

Comment:
You have to include this file in the index.rst of the documentation to make it present on https://integreat.github.io/integreat-cms/ (and to fix the `build-documentation` job of the CI pipeline):

#### `sphinx/index.rst`
```
...

Basic Concepts
==============

.. toctree::
    :caption: Basic Concepts
    :hidden:

    virtualenv
    internationalization
    documentation
    continuous-integration
    frontend-bundling

* :doc:`virtualenv`: Virtual Python environment and Pipenv
* :doc:`internationalization`: Multi-language support for the backend UI
* :doc:`documentation`: This Sphinx documentation
* :doc:`continuous-integration`: Continuous Integration and Delivery with CircleCI
* :doc:`frontend-bundling`: Bundling of static assets with Webpack

...
```

Reply:
I'm not able to test that locally 😞 
```
sed: -e: No such file or directory
sed: 1: "sphinx/ref-ext/cms.test ...": unterminated substitute pattern
sed: 1: "sphinx/ref-ext/sitemap.rst": unterminated substitute in regular expression

The patch for cms.rst could not be applied correctly.
Presumably the structure of the cms package changed.
Please adapt sphinx/patches/cms.diff to the structure changes.
```

Reply:
I added an entry, hopefully in the right way 😄